### PR TITLE
run modernize

### PR DESCRIPTION
**Pull request by nickpell**
@dimagi/py3

### CODE REVIEW

**Review by esoergel — COMMENTED** [review]
Happy with it aside from what Emord pointed out

File: corehq/celery_monitoring/heartbeat.py [hunk 1]
```diff
@@ -86,7 +87,7 @@ def heartbeat():
                 pass
             self.mark_seen()
 
-        heartbeat.func_name = self.periodic_task_name
+        heartbeat.__name__ = self.periodic_task_name
```
[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/celery_monitoring/heartbeat.py]: 24ed5e3bb5a

**Review by emord — APPROVED** [review]

File: corehq/celery_monitoring/heartbeat.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (esoergel) [reply corehq/celery_monitoring/heartbeat.py]: Is setting `heartbeat.func_name` completely redundant with `heartbeat.__name__` even in python 2?

[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/celery_monitoring/heartbeat.py]: I believe so:

```
>>> def x():
...     pass
... 
>>> x
<function x at 0x10a783398>
>>> x.func_name
'x'
>>> x.__name__ = 'y'
>>> x.func_name
'y'
>>> x.__name__
'y'
```

[the remark this answers is not in the document]
Comment (millerdev) [reply corehq/celery_monitoring/heartbeat.py]: Yes
```py
Python 2.7.15 (default, Sep 12 2018, 13:32:25)
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def foo():
...   pass
...
>>> foo.__name__
'foo'
>>> foo.func_name
'foo'
>>> foo.__name__ = "bar"
>>> foo.__name__
'bar'
>>> foo.func_name
'bar'
```

**Review by millerdev — APPROVED** [review]

File: corehq/celery_monitoring/heartbeat.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dannyroberts) [reply corehq/celery_monitoring/heartbeat.py]: Neat. Thanks!

**Review by dannyroberts — APPROVED** [review]

File: corehq/celery_monitoring/heartbeat.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nickpell) [reply corehq/celery_monitoring/heartbeat.py]: Turns out that setting ```__name__``` to unicode is forbidden in python 2: https://github.com/dimagi/commcare-hq/pull/23979

**Pull request merged**